src/MonticelloFileTree-Core/MCFileTreeAbstractReader.class.st

```diff
@@ -204,7 +204,7 @@ MCFileTreeAbstractReader >> packageProperties [
                                 on: Error
                                 do: [ :ex | 
                                     SystemNotification signal:
-                                                'Error reading package properties (.filetree): ' , packageDirectory pathName , ' :: ' , ex description ] ] ] ].
+                                                'Error reading package properties (.filetree): ' , packageDirectory asString , ' :: ' , ex description ] ] ] ].
```

Comment:
```suggestion
                                                'Error reading package properties (.filetree): ' , packageDirectory pathString , ' :: ' , ex description ] ] ] ].
```

I suggest to use #pathString instead because the print is more readable than #asString. In case of relative path, #asString will not print the full path for example